### ISSUE TITLE
Instructions for running unit tests obsolete in the README

### ISSUE THREAD

**Issue by JukkaL** [python/mypy#3891]
The README has these instructions which no longer work as expected:

```
For example, to run unit tests only, which run pretty quickly:

    $ ./runtests.py unit-test pytest
```

It currently runs no pytest tests (but this is just broken, which is a separate issue: #3890). Even if it would run all pytest tests, it would now also likely run integration tests, which are slow. Being able to run only unit tests is an important feature.

**Comment by JukkaL** [comment]
cc @elazarg

**Comment by JukkaL** [comment]
Fixed by #3894.
